RELEASENOTES.md

```diff
@@ -196,3 +196,11 @@
   various operations that rely on the `register_view` interface, such as the
   `dev_util.bank_regs` function and the `write-device-reg` and `probe-address`
   CLI commands.
+- `note 6` DMLC no longer emits warnings saying
```

Comment:
> will no longer emit the subset of `WEXPERIMENTAL` warnings saying...

maybe?

Reply:
I chose not to because I didn't want to bore the reader with details, but it does make sense to mention WEXPERIMENTAL because of how `--no-warn=WEXPERIMENTAL` is sprinkled in makefiles.

Reply:
Yeah that's mostly why I figured it was a good idea.